tests/utils/inputparams/testparams.go

```diff
@@ -42,10 +43,10 @@ func init() {
 func GetVolumeName() string {
 	return volumeName
 }
-
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volumeName string) string {
-	return volumeName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+func GetVolumeNameWithTimeStamp(volName string) string {
+	return volName + "_volume_" + strconv.FormatInt(rand.Int63(), 10)
```

Comment:
Any reason for this change? Please update comment at line 46 and remove line 49 if we decide to make this change.